src/components/organisms/Onboarding/StepActions.tsx

```diff
@@ -0,0 +1,42 @@
+import React, { FormEvent, ReactElement } from 'react'
+import Alert from '../../atoms/Alert'
+import Button from '../../atoms/Button'
+import Loader from '../../atoms/Loader'
+
+import styles from './StepActions.module.css'
+
+interface IStepAction {
+  buttonLabel: string
+  buttonAction: (e: FormEvent) => void
+  successMessage?: string
+  loading: boolean
+  completed: boolean
+  loadingMessage?: string
+}
+
+export default function StepActions({
```

Comment:
nitpick(non-blocking): Component always renders 1 action. It should be named `StepAction` isntead